api/cmd/db/migration.go

```diff
@@ -14,32 +14,63 @@ func Migrate(api *app.APIConfig) error {
 
 	// NOTE: when writing a migration for a new table, add the same in InitSchema
 	migration := gormigrate.New(api.DB(), gormigrate.DefaultOptions, []*gormigrate.Migration{
-		// NOTE: Add Migrations Here
+		{
+			ID: "202006071000",
+			Migrate: func(tx *gorm.DB) error {
+				if err := tx.AutoMigrate(&model.Tag{}, &model.Catalog{}, &model.Resource{}, &model.ResourceVersion{}).Error; err != nil {
```

Comment:
Are we missing &model.category ??

Reply:
Migration will run for creating new tables or columns in existing tables